Blockzilla/BrowserViewController.swift

```diff
@@ -1751,11 +1751,7 @@ extension BrowserViewController: KeyboardHelperDelegate {
         }
     }
 
-    func keyboardHelper(_ keyboardHelper: KeyboardHelper, keyboardDidHideWithState state: KeyboardState) {
```

Comment:
@rilury this was added by you to solve this bug https://github.com/mozilla-mobile/focus-ios/issues/2226 
Removing it will introduce the bug again. Maybe another solution will fit better.

Reply:
Hey, @Bogdan my solution was to move this functionality in the method textfieldDidEndEditing to avoid reintroducing the bug. This is why I also added the other changes and also tested the button from the ipad keyboard that removes the keyboard. Is this ok? Or should I add another change?

Reply:
@rilury Nice! This solution is ok and it solves the bug. Thanks!